streams/src/main/java/org/apache/kafka/streams/kstream/SessionWindows.java

```diff
@@ -69,6 +70,7 @@
  * @see KGroupedStream#windowedBy(SessionWindows)
  * @see TimestampExtractor
  */
+@SuppressWarnings("deprecation")
```

Comment:
Same here, I don't think we should need to suppress any deprecation warnings in this class as we aren't actually using any of the deprecated methods here, right? (And if we are, it's better to suppress at the smallest scope possible, ie only to specific individual methods rather than the whole class)

Reply:
Thanks @ableegoldman . It makes sense. I will fix that.